renderdoc/driver/vulkan/vk_resources.cpp

```diff
@@ -1348,6 +1348,8 @@ BlockShape GetBlockShape(VkFormat Format, uint32_t plane)
     case VK_FORMAT_B5G5R5A1_UNORM_PACK16:
     case VK_FORMAT_A1R5G5B5_UNORM_PACK16:
     case VK_FORMAT_B5G6R5_UNORM_PACK16:
+    case VK_FORMAT_A4R4G4B4_UNORM_PACK16_EXT:
```

Comment:
Nowhere better to put this comment - I think `A4R4G4B4` should be marked as BGRA order, the same as `R4G4B4A4` currently is.

The ResourceFormat struct can't differentiate between all four combinations but I *think* that should be handled as the old non-EXT versions were weird and were handled specially. But we should still list this one as a BGRA order format with `SetBGRAOrder`.